src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private int defaultSize = 10;
+    private int maxSize = 50;
+    private int maxBound = 5;
```

Comment:
The `defaultSize`, `maxSize`, and `maxBound` should be declared as constants (final static fields) with uppercase naming, according to Java naming conventions for constants.